airbyte-integrations/connectors/destination-gcs/src/test-integration/java/io/airbyte/integrations/destination/gcs/GcsDestinationAcceptanceTest.java

```diff
@@ -44,7 +48,8 @@
   protected static final Logger LOGGER = LoggerFactory.getLogger(GcsDestinationAcceptanceTest.class);
   protected static final ObjectMapper MAPPER = MoreMappers.initMapper();
 
-  protected final String secretFilePath = "secrets/config.json";
+  protected static final String SECRET_FILE_PATH = "secrets/config.json";
+  protected static final String SECRET_FILE_PATH_INSUFFICIENT_ROLES = "secrets/insufficientrolesconfig.json";
```

Comment:
Nitpick. Can you add underscores in the filename to make it more readable? Like `insufficient_roles_config.json`.

Reply:
I thought that our secret manager doesn't allow such symbols. Updated, thanks!